.travis.yml

```diff
@@ -10,11 +10,8 @@ cache:
     - node_modules
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.10
```

Comment:
Can we make this match:
https://github.com/ember-cli/ember-cli/blob/74562a4/blueprints/addon/files/.travis.yml#L15-L19